.obs/chartfile/operator/Chart.yaml

```
@@ -9,15 +9,15 @@ icon: https://raw.githubusercontent.com/rancher/elemental/main/logo/icon-element
 version: "%VERSION%"
 appVersion: "%VERSION%"
 annotations:
-  catalog.cattle.io/auto-install: elemental-crd=match
+  catalog.cattle.io/auto-install: elemental-operator-crd=match
```

```suggestion
  catalog.cattle.io/auto-install: elemental-operator-crds=match
```
I think it misses a `s`